src/util/server.c

```diff
@@ -234,7 +234,7 @@ int pidfile(const char *file)
 void orderly_shutdown(int status)
 {
 #if HAVE_GETPGRP
-    static int sent_sigterm;
+    static int sent_sigterm = 0;
```

Comment:
Did you get a compiler/Coverity warning here? I checked my 30 years old K+R book and it said `static` variables will be initialized to `0` if not set explicitly. I'm fine with improving readability here, but I wonder if there were other reasons?

Reply:
I did that for readability, It has no effect.